filament/src/materials/sao.mat

```diff
@@ -166,7 +170,7 @@ fragment {
         float ssRadius = max(1.0, tap.z * ssDiskRadius); // at least 1 pixel screen-space radius
 
         ivec2 ssSamplePos = ssOrigin + ivec2(ssRadius * tap.xy);
-        vec2 uvSamplePos = (vec2(ssSamplePos) + vec2(0.5)) * frameUniforms.resolution.zw;
+        vec2 uvSamplePos = (vec2(ssSamplePos) + vec2(0.5)) * materialParams.resolution.zw;
```

Comment:
Shoudn't UVs be in highp? Even at 1/4 res we will lose precision on some devices when the framebuffer is at native resolution.

Reply:
I think it's fine as long as we limit the size of the ssao buffer to 2048 -- which I'm not doing yet.